schema/transportation/segment.yaml

```diff
@@ -137,6 +210,193 @@ properties:
       enum: [ vehicle, motor_vehicle, car, truck, motorcycle, foot, bicycle, bus, hgv, hov, emergency ]
       "$comment": >-
         motor_vehicle includes car, truck and motorcycle
+    destinationSignSymbol:
+      description: >-
+        Indicates what special symbol/icon is present on a signpost, visible as road marking or similar.
+      type: string
+      enum: 
+      - motorway
+      - airport
+      - hospital
+      - center           # centre in raw OSM value
+      - industrial
+      - parking
+      - bus
+      - train_station
+      - rest_area
+      - ferry
+      - motorroad
+      - fuel
+      - viewpoint
+      - fuel_diesel
+      - food
+      - lodging
+      - info
+      - camp_site
+      - interchange
+      - toilets          # what's the appropriate US english value for this, restrooms?
+    road:
```

Comment:
You want us to go back to JSON land??

Reply:
ops, mistake at merge conflict resolution, reverted